Casks/invoker.rb

```diff
@@ -0,0 +1,27 @@
+cask "invoker" do
+  version "2.4.0"
+  sha256 "45babb26b0d1199803b23cd5eac8cfc711f27070851f1c5426daba827c64d49a"
+
+  url "https://invokerdev.fra1.digitaloceanspaces.com/invoker/Invoker-#{version}.dmg",
+      verified: "invokerdev.fra1.digitaloceanspaces.com/invoker/"
+  name "Invoker"
+  desc "Utility for managing Laravel applications"
+  homepage "https://invoker.dev/"
+
+  livecheck do
+    url "https://invokerdev.fra1.digitaloceanspaces.com/invoker/latest-mac.yml"
+    strategy :page_match
+    regex(/version:\s*(\d+(?:\.\d+)*)/i)
```

Comment:
I think this URL is an electron-builder appcast.
If so, you can simplify this to use:
```suggestion
    strategy :electron_builder
```
which should grab the `version` data.

Reply:
Wow! Didn't know that Brew supported this, that's great.